config/locales/en.yml

```diff
@@ -128,14 +128,21 @@ en:
           update_link: "%{docs_base_url}/api-documentation/activedocs-migrate"
           # end documentation links
   admin_sections:
-    partners: Accounts -- Applications
-    monitoring: Analytics
-    portal: Developer Portal
-    finance: Billing
-    settings: Settings
+    partners: 'Create, read, update and delete:<br/>
+      - developer <strong>accounts</strong><br/>
+      - <strong>applications</strong> of selected API products'
+    monitoring: 'Access & query <strong>analytics</strong> of:<br/>
+      - all API backends<br/>
+      - selected API products'
+    portal: 'Create, read, update, and delete content & code of the <strong>developer portal</strong>'
+    finance: 'Setup and manage <strong>customer billing</strong>'
+    settings: 'Update <strong>feature settings</strong>'
```

Comment:
I think "feature settings" can cause confusion with Plans Features, which are something else and currently NOT controlled by this permission flag. Examples of things under this flag:
- Developer Portal SSO config, 
- Some other "settings" pages under "Audience" (e.g., "Usage rules", "Fields definitions").

Reply:
How about 'settings pages' @guicassolato ? and adding this to member top description:
<img width="829" alt="Screenshot 2020-10-09 at 10 54 43" src="https://user-images.githubusercontent.com/54224/95563589-dd6a6980-0a1d-11eb-9532-c2c09504275b.png">

Reply:
Sounds good. Of course, we always have the option to rename the "Plan Features" to something else too. Anyway, whatever you think it would make more sense to the user.